scss/_alerts.scss

```diff
@@ -61,7 +61,9 @@
   padding: 15px;
   border-radius: 3px;
 
-  + .flash { margin-top: 5px; }
+  + .flash {
+    margin-top: 5px;
+  }
```

Comment:
The single-line properties are allowed by our linter actually, so no need to redo those.